selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/utils/BrowserLogsUtil.java

```diff
@@ -25,27 +27,115 @@
  * Read and store browser logs to the test logs. Log level and type are defined in {@link
  * org.eclipse.che.selenium.core.SeleniumWebDriver#doCreateDriver(URL)}
  */
-@Singleton
 public class BrowserLogsUtil {
```

Comment:
Class name is outdated so as it is not about just BROWSER logs, but PERFORMANCE logs as well. So, lets make the name more general: `WebDriverLogsReader`